caffeine/src/main/scala/akka/caffeine/internal/CaffeineFanOut2Stage.scala

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright (C) 2016 Lightbend Inc. <http://www.lightbend.com>
+ */
+package akka.caffeine.internal
+
+import akka.caffeine.Aggregator
+import akka.stream.{ FanOutShape2, _ }
+import akka.stream.stage.{ GraphStage, GraphStageLogic, InHandler, OutHandler }
+
+import scala.language.postfixOps
```

Comment:
I don't think that should be used more than necessary, i.e. only when it adds some substantial readability value.